bls/src/main/java/tech/pegasys/teku/bls/BLS.java

```diff
@@ -43,7 +44,17 @@
 
   private static final Logger LOG = LogManager.getLogger();
 
-  private static BLS12381 BlsImpl = MikuliBLS12381.INSTANCE;
+  private static BLS12381 BlsImpl;
```

Comment:
nit: 
```suggestion
  private static final BLS12381 blsImpl;
```

Also see comment on `MikuliBLS12381.INSTANCE`.

Reply:
Good, made it final. Though I was going to make the implementation pluggable at runtime, but this would be another PR
> Also see comment on MikuliBLS12381.INSTANCE

Did you mean `BlstBLS12381.INSTANCE` ?

Reply:
Yes I did sorry.  About being careful with loading the native lib as part of class initialisation.